lms/static/certificates/vendor/bi-app/_variables-ltr.scss

```diff
@@ -0,0 +1,15 @@
+// ------------------------------------------
+// left to right variables to be used by bi-app mixins
+// authors: 
+// twitter.com/anasnakawa
+// twitter.com/victorzamfir
+// licensed under the MIT license 
+// http://www.opensource.org/licenses/mit-license.php
+// ------------------------------------------
+
+// namespacing variables with bi-app to
+// avoid conflicting with other global variables
+$bi-app-left 				    : left;
+$bi-app-right 				  : right;
+$bi-app-direction 			: ltr;
+$bi-app-invert-direction: rtl;
```

Comment:
Hmm, this is a vendor file (bi-app in this case). I'll see what we do in our platform's version and mirror that here. Thanks for the heads up.